tools/cve/base-images.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2022 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Eeo pipefail
+shopt -s failglob
+
+if [[ "x$SEVERITY" == "x" ]]; then
+  SEVERITY="CRITICAL,HIGH"
+fi
+
+function __base_images_tags__ {
```

Comment:
We can add a comment that this is a hack before we figure out how to store base images in the repo.